tools/sgen/input/FluentUIStyle.yml

```diff
@@ -161,3 +162,14 @@ SecondaryButtonTokens extends MSFButtonTokens:
 GhostButtonTokens extends MSFButtonTokens:
   borderColor: [ rest: $Colors.Neutral.clear, hover: $Colors.Neutral.clear, pressed: $Colors.Neutral.clear, selected: $Colors.Brand.shade10, disabled: $Colors.Neutral.clear ]
 
+AP_MSFListTokens:
+  backgroundColor: $Colors.Background.neutral1
+  borderColor: $Colors.Stroke.neutral1
+  borderSize: $Border.size.thin
+  padding: $Spacing.medium
+  textFont: $Typography.body
+  subtitleFont: $Typography.subheadline
+  iconColor: [ iconOnly: $Colors.Foreground.neutral3, withLabel: $Colors.Foreground.neutral4, disclosure: $Colors.Foreground.neutral4 ]
```

Comment:
I think the variations in the figma should be appearance proxies that extend MSFListTokens.
Once we get the state colors like pressed, hover, etc... this will be a matrix of styles vs states for this color.

Reply:
https://github.com/microsoft/fluentui-apple/issues/362